communicator/service_connect_communicator.cpp

```diff
@@ -0,0 +1,48 @@
+#include "service_connect_communicator.h"
+
+using namespace std;
+
+bool Service_Connect_Communicator::connect(const string& address)
+{
+    ACE_INET_Addr addr(SERV_PORT, (address.c_str()));
```

Comment:
```suggestion
    ACE_INET_Addr addr(SERV_PORT, address.c_str());
```
不需要给address.c_str()额外加括号，操作符的优先级请牢记